system/jhtree/jhtree.cpp

```diff
@@ -1042,6 +1041,28 @@ void CKeyStore::clearCacheEntry(const IFileIO *io)
     }
 }
 
+//
+
+static void noteCLSeeks(IContextLogger *ctx, unsigned lseeks, unsigned lscans, unsigned lwildseeks)
```

Comment:
Trivial: Do these need CL in the name (I didn't immediately understand what it meant).